Gemfile

```diff
@@ -1,3 +1,4 @@
 source "http://rubygems.org"
 gemspec
 
+gem 'activerecord', ENV['RAILS_VERSION']
```

Comment:
The one in the gemspec will have to be redefined so that its not requiring minumum `4.0`.